api/opds.py

```diff
@@ -721,6 +721,10 @@ def language_and_audience_key_from_work(self, work):
             audiences = Classifier.AUDIENCES_JUVENILE
         elif work.audience in Classifier.AUDIENCES_ADULT:
             audiences = list(Classifier.AUDIENCES)
+        elif work.audience == Classifier.AUDIENCE_ALL_AGES:
+            audiences = [Classifier.AUDIENCE_ALL_AGES]
```

Comment:
I think this should also include AUDIENCE_CHILDREN, but not AUDIENCE_ADULT.

Reply:
I don't understand why this that's the case for this.

Reply:
Here's my thinking:  we never want to show books that are a jump up in age level. In this case, if you are in the children's section and you look at an all-ages book by Conan Doyle, then ask for more books by the same author, we don't want to show his audience="Adult" books. But we do want to show his audience="Children" books, not just the other "All Ages" stuff.

This is the same logic behind showing all of the audiences when you're looking at works by this author for audience="Adult". It's okay to show an All Ages book in that context, and even a Children book, because there's no risk of showing age-inappropriate material. But not vice versa.

Reply:
Thanks, that helps.